RELEASE.md

```diff
@@ -130,20 +130,30 @@ To release a new version of Spyder you need to follow these steps:
 
 ### Check release candidate
 
-* Update version in `__init__.py` (set release version, remove 'dev0', add 'preX'), then
+* Update version in `__init__.py` (set release version, remove '.dev0', add 'rcX'), then
 
       git add .
-      git commit -m "Release X.X.XpreX [ci skip]"
+      git commit -m "Release X.X.XrcX [ci skip]"
       git push upstream 5.x
 
 * Manually activate the following workflows (see [Running a workflow](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#running-a-workflow)):
-    - Create Windows Installer
-    - Create macOS App Bundle and DMG
-    - Create conda-based installers for Windows, macOS, and Linux
+    - Create Windows Installer *
+    - Create macOS App Bundle and DMG *
+    - Create conda-based installers for Linux
+
+      **Note:** For the Windows and macOS installers you will need to trigger the workflows through the [GitHub REST API](https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event) or using the [GitHub CLI](https://cli.github.com/manual/gh_workflow_run) (the GitHub CLI is available at https://cli.github.com/). In case the GitHub CLI is used you will need to run:
```

Comment:
```suggestion
      **Note:** For the Windows and macOS installers you need to trigger their workflows through the [GitHub REST API](https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event) or using the [GitHub CLI](https://cli.github.com/manual/gh_workflow_run) (the GitHub CLI is available at https://cli.github.com/). In case the GitHub CLI is used, you need to run:
```